test/built-ins/String/prototype/isWellFormed/prop-desc.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2020 Rick Waldron. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-string.prototype.is-well-formed
+description: >
+  Property type and descriptor.
+info: |
+  String.prototype.isWellFormed( )
+
+  17 ECMAScript Standard Built-in Objects
+includes: [propertyHelper.js]
+features: [String.prototype.isWellFormed]
+---*/
+assert.sameValue(typeof String.prototype.isWellFormed, 'function');
+
+assert.sameValue(
+  typeof String.prototype.isWellFormed,
+  'function',
```

Comment:
Duplicate test

Reply:
actually prop-desc seems the appropriate place to test the type of the value.